.pre-commit-config.yaml

```diff
@@ -0,0 +1,39 @@
+# Configuration file for pre-commit (https://pre-commit.com/).
+# Please run `pre-commit run --all-files` when adding or changing entries.
```

Comment:
Can you run `pre-commit autoupdate` to ensure all these hooks are up-to-date w/ their latest versions?

Reply:
yep. done.